awx/settings/defaults.py

```diff
@@ -386,9 +386,6 @@ def IS_TESTING(argv=None):
 # when trying to access a UI page that requries authentication.
 LOGIN_REDIRECT_OVERRIDE = ''
 
-# If set, serve only minified JS for UI.
-USE_MINIFIED_JS = False
```

Comment:
👍 I don't see any evidence of this; we always do this for builds now, I assume cc @jakemcdermott @mabashian